bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,119 @@
+<mujoco model="cf2">
+  <option integrator="RK4"/>
+  <compiler inertiafromgeom="false" coordinate="local" meshdir="cf2"/>
+  <default>
+    <default class="visual">
+      <geom group="2" type="mesh" contype="0" conaffinity="0"/>
+    </default>
+    <default class="collision">
+      <geom group="3" type="mesh"/>
+    </default>
+  </default>
+  <asset>
+    <material name="Material10-polished_plastic1_001" specular="0.5" shininess="0.225"
```

Comment:
Can you give the material more readable names?

* `Material10-polished_plastic1_001` -> `polished_plastic`

Also, can you format the material rgbas to no more than 2-3 decimal places?